3rdparty/libogg/wscript

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env python
+# encoding: utf-8
+
+FRAGMENT_INTTYPES_H='''#include <inttypes.h>
```

Comment:
If you're checking for a singular header, there is `header_name` arg in check_c.

Reply:
Then you also don't need to supply `msg` arg, it will generate it from header_name. 

I actually don't remember if it's called header_name and can't check it right now.